src/qa4sm_reader/plotter.py

```diff
@@ -673,10 +682,9 @@ def barplot(
                                          df=values,
                                          type='barplot_basic',
                                          Var=Var)
-            if other_meta:
-                out_name = self.create_filename(Var, type='barplot_basic_3ds')
-            else:
-                out_name = self.create_filename(Var, type='barplot_basic')
+            out_name = globals.barplot_fn
```

Comment:
again here, would it be better to use the `self.create_filename function`?

Reply:
done.